projects/aca-content/src/lib/components/search/search-results-row/search-results-row.component.ts

```diff
@@ -114,4 +151,10 @@ export class SearchResultsRowComponent implements OnInit, OnDestroy {
     event.stopPropagation();
     this.store.dispatch(new NavigateToFolder(this.node));
   }
+
+  private stripHighlighting(highlightedContent: string): string {
+    return highlightedContent && highlightedContent !== ''
```

Comment:
'' is falsy so you don't need second condition (highlightedContent !== '' -> this can be removed) because first condition will allow for truthy only

Reply:
Right, fixed